### PR TITLE
FIR UAST: test updates (property accessor annotations)

### PR DESCRIPTION
**Pull request by jsjeon**
This is simple test data update for [KOTLIN-CR-183](https://kotlin.jetbrains.space/p/kotlin/reviews/183/). It's atop #1690 and only the last commit matters.

### CODE REVIEW

File: plugins/kotlin/uast/uast-kotlin-fir/testData/legacyTypes/SimpleAnnotated.types.fir.txt [hunk 1]
```diff
@@ -15,8 +15,5 @@ UFile (package = ) [public final class SimpleAnnotated {...]
                     UIdentifier (Identifier (println)) [UIdentifier (Identifier (println))]
                     USimpleNameReferenceExpression (identifier = println, resolvesTo = null) [println] : PsiType:Unit
```
Comment (jsjeon) [inline plugins/kotlin/uast/uast-kotlin-fir/testData/legacyTypes/SimpleAnnotated.types.fir.txt:16]: This `Unit` v.s. `void` in FE 1.0 UAST is only difference for type test. That is, I expect #1690 can make this identical too. Whichever goes first, I can rebase the remaining one.

Comment (jsjeon) [reply plugins/kotlin/uast/uast-kotlin-fir/testData/legacyTypes/SimpleAnnotated.types.fir.txt:16]: Okay, maybe bigger ones can go first. :)

**Review by darthorimar — APPROVED** [review]

File: plugins/kotlin/uast/uast-kotlin-fir/testData/legacyTypes/SimpleAnnotated.types.fir.txt (hunk 1 above)
Comment (jsjeon) [reply plugins/kotlin/uast/uast-kotlin-fir/testData/legacyTypes/SimpleAnnotated.types.fir.txt]: Yup, indeed identical on top of #1690

**Comment by darthorimar** [comment]
Merged, thank you!

**Pull request closed without merging**